dspace/config/registries/schema-person-types.xml

```diff
@@ -0,0 +1,68 @@
+<dspace-dc-types>
+
+    <dspace-header>
+        <title>Person Types</title>
+    </dspace-header>
+
+    <dc-schema>
+        <name>person</name>
+        <namespace>https://schema.org/Person</namespace>
+    </dc-schema>
+
+
+    <!--NO EQUIVALENT FOUND
+        - person.identifier.orcid > NOT FOUND
```

Comment:
Use dc.identifier.uri?

Reply:
It's indeed an option to use dc.identifier.uri and store e.g. https://orcid.org/0000-0002-3577-2890
This would imply the only way to determine it's an ORCID is based on the "https://orcid.org/" prefix
Hopefully we get more info from the communication with the schema.org team

Reply:
And the Person:identifier (PropertyValue  or Text  or URL )? «The identifier property represents any kind of identifier for any kind of Thing, such as ISBNs, GTIN codes, UUIDs etc. Schema.org provides dedicated properties for representing many of these, either as textual strings or as URL (URI) links. »

Reply:
https://schema.org/docs/datamodel.html#identifierBg
«"identifier" property
Background information on the 'identifier' property and its sub-properties.

The identifier property and its sub-properties are primarily useful in cases where the content is expressed as a textual string. Increasingly there are canonical URL/URI representations for each of these. All schema.org syntaxes already have built-in representation for URIs and URLs, e.g. in Microdata 'itemid', in RDFa 1.1, 'resource', in JSON-LD, '@id'. Generally it is preferable to use these unless there is a specific requirement to explicitly state the kind of identifier, or to provide additional / alternative identifiers (e.g., DOIs). Such requirements are common e.g. for scientific dataset description.

In some cases the values of the identifier property indicate a set of (somewhat) interchangeable entities rather than a single distinct real world entity. Such sets could be viewed as corresponding to classes, but we do not explore that possibility here. For example sku and the various product-related GTIN codes. However identifier is not intended to cover broader categorization and classification mechanisms. For example, although the isicV4 property has values that are in some sense identifiers, we do not treat isicV4 as a subproperty of identifier since it serves to identify a category not an individual Thing (specifically, Person). Similarly, very many schema.org properties can have values that are written as URLs, but we do not treat those properties as specializations of identifier.

In the most complex case, there is sometimes a need to represent the type of an identifier. In this case, a PropertyValue pair ('name', 'identifier') pair can be used when a standard URI form of the identifier is unavailable. We do not currently have a recommended identifier scheme for identifier schemes, but in most cases there is a conventional short name for most identifier schemes (which should be used in lowercase form).»